Source/Steam/emulator.json

```diff
@@ -0,0 +1,12 @@
+{
+  "emulator_v0": {
+    "argv": "./usr/bin/FEX",
+    "environment": { "FEX_PORTABLE": "1" },
+    "container_argv": "./usr/bin/FEX",
+    "container_environment": { "FEX_ROOTFS": "" },
+    "main_argv": "./FEXCompatTool",
```

Comment:
It's `FEXCompatTool` here, but `fex-compat-tool` in the build system when you compile it. Don't the two names need to be the same?

Reply:
Oops.